samza-core/src/main/java/org/apache/samza/config/JobConfig.java

```diff
@@ -47,6 +47,15 @@
   public static final String CONFIG_REWRITERS = "job.config.rewriters";
   public static final String CONFIG_REWRITER_CLASS = "job.config.rewriter.%s.class";
 
+  /**
+   * job.config.loader.factory specifies {@link ConfigLoaderFactory} to get {@link ConfigLoader}
+   */
+  public static final String CONFIG_LOADER_FACTORY = "job.config.loader.factory";
+  /**
+   * job.config.loader.properties prefix wraps properties needed for {@link ConfigLoader} to loader config.
```

Comment:
Do you think it would be helpful to put a small example here about how the prefix will be used?

Reply:
Added